ecms-social-integration/src/test/java/org/exoplatform/ecms/activity/processor/ActivityAttachmentProcessorTest.java

```diff
@@ -0,0 +1,56 @@
+package org.exoplatform.ecms.activity.processor;
+
+import org.exoplatform.services.wcm.core.NodeLocation;
+import org.exoplatform.social.core.activity.model.ExoSocialActivity;
+import org.exoplatform.social.core.activity.model.ExoSocialActivityImpl;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import javax.jcr.Node;
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(NodeLocation.class)
+public class ActivityAttachmentProcessorTest {
+
+    @Mock
+    private ActivityAttachmentProcessor activityAttachmentProcessor;
+
+    @Test
+    public void processActivity() {
+        ExoSocialActivity activity = new ExoSocialActivityImpl();
+        activity.setType("sharefiles:spaces");
+        HashMap<String, String> templateParams = new HashMap<>();
+        templateParams.put("workspace", "collaboration");
+        templateParams.put("author", "root");
+        templateParams.put("permission", "read");
+        templateParams.put("nodePath", "/Groups/spaces/new/Documents/Shared/sample.pdf");
+        templateParams.put("docTitle", "sample.pdf");
+        templateParams.put("mimeType", "application/pdf");
+        templateParams.put("message", "");
+        templateParams.put("repository", "repository");
+        templateParams.put("contentName", "sample.pdf");
+        templateParams.put("nodeUUID", "736ec0d07f0001015e91bad4c1d582e1");
+        templateParams.put("id", "736ec0d07f0001015e91bad4c1d582e1");
+        activity.setTemplateParams(templateParams);
+        NodeLocation nodeLocation = new NodeLocation(templateParams.get("repository"), templateParams.get("workspace"),
+                templateParams.get("nodePath"), templateParams.get("nodeUUID"), true);
+        Node currentNode = mock(Node.class);
+        doCallRealMethod().when(activityAttachmentProcessor).processActivity(activity);
+        mockStatic(NodeLocation.class);
```

Comment:
```suggestion
        PowerMockito.mockStatic(NodeLocation.class);
```

This should prevents you to use mock-inline. I prefer to not change mockito version for the moment
Can you try with this ? 
If problem with your test, ping me

Reply:
Okay i will try it

Reply:
Yes it's working working without mockito-inline